### DISCUSSION TITLE
GIT repo sync fails: this.queue is not a function

### DISCUSSION THREAD

**Discussion by rudesome** [requarks/wiki#4405] [Error / Bug Report] [answered]
I get the following error message when i try to sync to my GIT repo:

![image](https://user-images.githubusercontent.com/26667983/132978479-c797982d-183e-42a4-9f07-744cdb83146a.png)

This issue started after the last update a couple of hours ago.
![image](https://user-images.githubusercontent.com/26667983/132978697-ebd3b55e-e679-4320-864e-5890a9f0f393.png)

**Comment by EricFromCanada** [comment]
The issue appears to lie with #3821. Rolling back `server/core/scheduler.js` to the [previous version](https://github.com/Requarks/wiki/blob/71aa0c9346c3a120c4c1ca4c98c86c949a5f6b2f/server/core/scheduler.js) will fix it for now.

**Reply by AntonAndreevichMoroz** [reply]
Thanks. It helped me. We will wait for an update and a solution to this problem.

**Reply by MrakoMaks** [reply]
> The issue appears to lie with #3821. Rolling back `server/core/scheduler.js` to the [previous version](https://github.com/Requarks/wiki/blob/71aa0c9346c3a120c4c1ca4c98c86c949a5f6b2f/server/core/scheduler.js) will fix it for now.

Thanks. It helped me to.

**Reply by jesgp** [reply]
Helped me too - thanks for the information

**Reply by AVTPJ** [reply]
Yep, this worked.

**Comment by EricFromCanada** [comment, marked as answer]
Fix proposed in https://github.com/Requarks/wiki/pull/4481.
